twitter_downloader/utils.py

```
@@ -375,3 +376,106 @@ def get_tweet_id_from_url(tweet_url: str) -> str:
+
+        try:
+            response = requests.get(
+                self.API_URL,
+                params={"tweet": tweet_url},
+                headers=headers,
+                timeout=30,
+            )
+            print("++++++arter", response.status_code, response.text)
+            response.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            logger.error(f"Network error when connecting to Twitter API v4: {str(e)}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)